jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeSpatPayload.java

```diff
@@ -38,7 +38,7 @@ public J2735SPAT getSpat() {
 	        return (J2735SPAT) getData();
 	    }
 
-	    public void setBsm(J2735SPAT spat) {
+	    public void setSpat(J2735SPAT spat) {
```

Comment:
Good catch, but this seems like a bug fix rather part of the MAP based update

Reply:
Yeah, I found this bug while I was implementing MAP. Update the description to include fixing the spat payload method naming